packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.tsx

```diff
@@ -55,12 +55,18 @@ const AccountOption = ({ account, idx }: TAccountsList) => {
             </div>
 
             <span className='account-transfer-form__balance'>
-                <Money
-                    amount={account.balance}
-                    currency={account.currency}
-                    has_sign={Boolean(account.balance && Number(account.balance) < 0)}
-                    show_currency
-                />
+                {is_poa_poi_pending_for_mf && is_selected_from ? (
```

Comment:
```suggestion
                {is_pending_verification && is_selected_from ? (
```
We'd probably move `AccountOption` out of `account-transfer-form.tsx`. I don't think `AccountOption` need to know business logic like is it poa/poi pending for mf etc. Imo, it just needs to know if it's pending verification or not.

Reply:
ahhh sure sure 👍